### DISCUSSION TITLE
Parallelism

### DISCUSSION THREAD

**Discussion by davidfrankovertica** [terramate-io/terramate#1368] [Q&A] [answered]
The following quite from the [README.md](https://github.com/terramate-io/terramate?tab=readme-ov-file#what-are-the-most-common-use-cases-of-terramate-cli) file states the **parallel deployment** option when using the terramate-cli, but I cannot see any option to use this in CLI. Is this planned for the future or is it just a typo in documentation?

> Stacks unlock faster execution, better collaboration, and lower blast radius risk. With stacks, parallel deployment ensures no more pipeline run bottlenecks.

Regardless, I do believe this part should be removed from the documentation if CLI tool does not support it, as it is misleading.

**Comment by soerenmartius** [comment, marked as answer]
Hi @davidfrankovertica,

Thanks for reaching out! We are indeed working on parallel execution. The challenge is to make outputs readable to the user when executing commands in parallel. Indeed, we shouldn't mention this in the documentation yet, and I will update it accordingly.

Let me get back to you once I have an ETA for parallel execution.

Again, thanks for reporting this.

**Comment by soerenmartius** [comment]
We added parallel execution some weeks ago :) Sorry I forgot to update this Discussion. Please find the docs for `--parallel` at https://terramate.io/docs/cli/orchestration/#parallel
